src/components/EditContactInformation.jsx

```diff
@@ -0,0 +1,129 @@
+import { FormControl, FormLabel, Input, Heading, Button } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { useBackend } from '../contexts/BackendContext';
+const EditContactInformation = () => {
+  const BUSINESS_ID = 1; // hard coded business_id constant for development purposes
+  const { backend } = useBackend();
+
+  const [businessContactInfo, setBusinessContactInfo] = useState({
+    phoneNumber: '',
+    email: '',
+    website: '',
+    street: '',
+    firstName: '',
+    lastName: '',
+  });
+
+  const handleChange = event => {
+    const name = event.target.name;
+    var value = event.target.value;
+
+    setBusinessContactInfo(prevState => ({ ...prevState, [name]: value }));
+  };
+
+  const updateContactInfo = async () => {
+    try {
+      await backend.put(`/business/${BUSINESS_ID}`, {
+        contact_name: `${businessContactInfo.firstName} ${businessContactInfo.lastName}`,
+        contact_phone: businessContactInfo.phoneNumber,
+        primary_email: businessContactInfo.email,
+        website: businessContactInfo.website,
+        street: businessContactInfo.street,
+      });
+    } catch (error) {
+      console.error('Error updating data:', error);
+    }
+  }
+
+  useEffect(() => {
+    const getBusinessContactResponse = async () => {
+      try {
+        const businessContactResponse = await backend.get(`/business/${BUSINESS_ID}`);
+        const businessContact = businessContactResponse.data[0];
+        const name = businessContact.contact_name.split(' ');
+        const firstName = name[0];
+        const lastName = name[1];
+        const phoneNumber = businessContact.contact_phone;
+        const email = businessContact.primary_email;
+        const website = businessContact.website;
+        const street = businessContact.street;
```

Comment:
Instead of creating these variables, you can call those lines of code inside of setBusinessContactInfo
example: `phoneNumber: businessContact.contact_phone` instead of `phoneNumber: phoneNumber`
However, keep the firstName and lastName as it is currently